community/io/src/main/java/org/neo4j/io/pagecache/impl/muninn/MuninnPage.java

```diff
@@ -84,6 +91,32 @@ public long address()
         return pointer;
     }
 
+    public long getLastModifiedTxId()
+    {
+        return lastModifiedTxId;
+    }
+
+    public long resetLastModifiedTxId()
```

Comment:
Would be nice if the name mentioned that it returned the txId, like `getAndResetLastModifiedTxId`